server.py

```diff
@@ -110,29 +110,34 @@ def get(self, path, feature_id=None):
         feature_list = self.__get_feature_list()
         return common.JSONHandler.get(self, feature_list, formatted=True)
       elif path.endswith('.xml'): # Atom feed request.
-        filterby = None
-
-        category = self.request.get('category', None)
-
-        # Support setting larger-than-default Atom feed sizes so that web
-        # crawlers can treat use this as a full site feed.
-        try:
-          max_items = int(self.request.get('max-items',
-                                           settings.RSS_FEED_LIMIT))
-        except TypeError:
-          max_items = settings.RSS_FEED_LIMIT
-
-        if category is not None:
-          for k,v in models.FEATURE_CATEGORIES.iteritems():
-            normalized = normalized_name(v)
-            if category == normalized:
-              filterby = ('category =', k)
-              break
-
-        feature_list = models.Feature.get_all( # Memcached
-            limit=max_items,
-            filterby=filterby,
-            order='-updated')
+        status = self.request.get('status', None)
```

Comment:
Hmm, not sure. I think I was on a slow connection and this wasn't intended for here. Ignore.